src/DesignMyNight/Elasticsearch/QueryGrammar.php

```diff
@@ -158,13 +158,16 @@ protected function compileWhereBasic(Builder $builder, array $where): array
             '<=' => 'lte',
         ];
 
-        if (is_null($value)) {
+        if (is_null($value) || $where['operator'] == 'exists') {
```

Comment:
exists should probably be a constant.  I know it's unlikely but if ES change it to be something like 'does_exist' then it'll be covered.

Reply:
This is the only place it would be checked so I don't know that a constant helps us at all here?If ES changed their grammar then we'd need to update lots of things in QueryGrammar to match their new format, so I don't think a constant would add anything.

Reply:
fair poiint